Headstone update

RadarStatic opened the pull request:
Fuckin' hate this code. So much. Added tombstones as decals, and put in Tk's weapon sprites, with one example weapon (that doesn't work whatsoever, but will compile.)

RadarStatic committed:
Headstone update

Fuckin' hate this code. So much.

TalkingCactus commented on code/dallus_content/dallus_content/mob/shadowbeast.dm:
```diff
@@ -1,10 +1,10 @@
+	icon_living = "shadowbeast"
+	icon_dead = "shadowbeast_dead"
 
 	faction = "carp"
```
🤔

Tk420634 replied:
So does this work... or?

RadarStatic committed:
Added Killbow's Weapons

Added, coded and unfucked a number of Killbow's sprites. They still need sprites for on the back/belt slots, however.
